deploy/vagrant/Vagrantfile

```diff
@@ -46,7 +51,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           is_vb = Provider == "vb"
           tachyon_dist = UfsV.tachyon_dist(tachyon_version)
           puts "using tachyon distribution #{tachyon_dist}"
+          tachyon_platform = "local"
```

Comment:
What does "local" mean? For "local" will this actually deploy on my local machine? What about ec2?